applications/prompt-keda-lsstcam/values-usdfprod-prompt-processing.yaml

```diff
@@ -0,0 +1,118 @@
+prompt-keda:
+  image:
+    pullPolicy: IfNotPresent
+    tag: 6.4.0
+
+  worker:
+    # TODO: need to adjust this once we know how leaky the LSSTCam pipeline is
+    # restart: 7
+    # Expect much less memory usage/leakage for Single-Frame only
+    restart: 30
+
+  instrument:
+    pipelines:
+      # IMPORTANT: don't use flow-style mappings (i.e., {}) in pipelines specs
+      # if the result (including any comments) is longer than 72 characters.
+      # The config will get corrupted after template substitution.
+      # Block-style mappings can have lines of any length.
+      main: |-
+        - survey: BLOCK-T407  # FBS SV Field Survey
```

Comment:
From Keith's talk back in JTM, I thought that all in-focus science blocks will have `BLOCK-365`?  Did the plan change?  I likely missed to follow more recent discussions...

Reply:
From slack search, it sounds like there was a meeting where this was discussed today and @ebellm  was there; maybe he knows?

Reply:
Oops! And that was even in my notes.

I'll move BLOCK-T407 to the "manual override" section, assuming their relationship is analogous to BLOCK-320 vs. BLOCK-T155 for ComCam.

Reply:
The discussion didn't fully converge, but yes I think BLOCK-365 is the correct default for now.